scripts/provisioning/s3_start

```diff
@@ -58,6 +61,10 @@ def main():
     elif args.service == 's3bgconsumer':
       # Add action and remove pass
       pass
+    elif args.service == 'haproxy':
+      # TODO: logging path has to come from conf store -- to be fixed during
+      # end-to-end logging integration.
+      os.system("/usr/sbin/haproxy -Ws -f /etc/cortx/s3/haproxy.cfg -p /run/haproxy.pid 2>&1 1>/var/log/cortx/s3/haproxy.log")
```

Comment:
do we need to create this directory  first ? 
/var/log/cortx/s3/haproxy

Reply:
We are doing it at line 50.